dapps.asciidoc

```diff
@@ -164,16 +164,17 @@ Github link; https://github.com/embark-framework/embark
 
 Website link; https://github.com/embark-framework/embark
 
-[[dapp_develotment_tool_sec]
-==== DApp (development tool)
-DApp is a simple command line tool for smart contract development. It supports these common usecases:
+[[dapp_develotment_tool_sec]]
```

Comment:
I do not know if there is a criterion admitted in the book for the denomination of a **Decentralized App**, I can not say whether we should use **DApp** or **Dapp**.

Reply:
Here is a typo **"develotment"** -> **"development"**.